.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: ruby
+script: "bundle exec rspec spec"
+notifications:
+  email: false
+rvm:
+  - 2.3.1
```

Comment:
2.3.1 is not a ruby version that we use at puppet. It's either 2.1.9 for puppet4, or 2.4.1 for puppet 5.